Handlers/AuthenticationHandler.cs

```diff
@@ -58,6 +61,24 @@ public async Task<string> Authenticate()
             return Token;
         }
 
+        public async Task<string> Authenticate(string pat)
+        {
+
+            var _address = ApiHelper.Instance.SiteUrl + string.Format(ApiHelper.Instance.GetAllEmployeeEndpoint, 1);
+
+            try
+            {
+                string _jsonResult = ApiHelper.Instance.WebClient(pat).DownloadString(_address);

```

Comment:
@BrianMoos If you catch WebException and run the response through ApiHelper.Instance.ProcessApiResponseContent you  will get better error message. And will get a 401 if the PAT is invalid. 
Keep the general catch and change the message.

Reply:
Thanks,
I don't understand why `ProcessApiResponseContent` needs `responseContent` as a parameter as it can be extracted from `webEx`. What am I missing?

Reply:
Hmm... I can't remember if there was a reasoning for this in the initial versions. 
But you're right. Extraction could be centralized in the ProcessApiResponseContent method as it is now.